frontend/src/components/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types'
+
+const Button = ({
+  href,
+  text,
+  className,
+}: {
+  text: string
+  href: string
+  className: string
+}) => {
+  return (
+    <div>
+      <a href={href} className={className}>
+        {text}
+      </a>
+    </div>
+  )
+}
+
+Button.defaultProps = {
+  text: 'Button',
+}
```

Comment:
Please use an interface to set up props, using default props will not allow intellisense to detect invalid props. Additionally, there are more props in this component that have not been listed and as a result can cause errors if they are not defined and the app is expecting them